src/Shared/LayoutRenderers/AspNetTraceIdentifierLayoutRenderer.cs

```diff
@@ -23,7 +22,20 @@ protected override void DoAppend(StringBuilder builder, LogEventInfo logEvent)
             builder.Append(LookupTraceIdentifier(httpContext));
         }
 
-#if ASP_NET_CORE
+        /// <summary>
+        /// Ignore the System.Diagnostics.Activity.Current.Id value (AspNetCore3 uses ActivityId by default)
+        /// </summary>
+        public bool IgnoreActivityId { get; set; }
```

Comment:
What about PreferActivityId (and then true by default?)

Reply:
I like to keep the "action" words to a minimum for consistency (Enable / Disable / Ignore). And when something is enabled by default then I prefer the setting to have opposite name.

Reply:
I prefer giving flags never negative names (so that you have false+false=true), but OK, it's not that important for here.